client/landing/gutenboarding/index.tsx

```diff
@@ -30,7 +32,14 @@ window.AppBoot = () => {
 
 		ReactDom.render(
 			<BrowserRouter basename="gutenboarding">
-				<Gutenboard />
+				<Switch>
+					<Route exact path={ path }>
+						<Gutenboard />
+					</Route>
+					<Route>
+						<Redirect to={ Step.IntentGathering } />;
```

Comment:
This is a relative path now (`Step.IntentGathering` is literally `'about'`). That means that e.g. `/pages/foobar` now redirects to `/pages/about` :sweat_smile: 

We should fix this so it becomes an absolute redirect. If we can't easily use `usePath` here, let's maybe use `react-router`'s `generatePath` and `path` from our own `path.ts` file.

Reply:
Good catch!

Reply:
Fixed! https://github.com/Automattic/wp-calypso/pull/39312/commits/09131c7df6ce3360c7b6433f5ad3e4d2d9f2dda0